app/services/catalog/collect_local_tag_resources.rb

```diff
@@ -0,0 +1,37 @@
+module Catalog
+  class CollectLocalTagResources
```

Comment:
If we move the namespace above, this could be renamed to something that references an order e.g. `CollectOrderResources`.